lib/commands/packages.js

```diff
@@ -131,7 +131,7 @@ module.exports = async (defs) => {
       const from = allPackagesBeforeChanges[name];
       const to   = allPackagesAfterChanges[name];
 
-      console.log(`PACKAGE-UPGRADED: ${ from } -> ${ to }`);
+      console.log(`PACKAGE-UPGRADED: ${ name }(${ from }) -> ${ name }(${ to })`);
```

Comment:
what about instead of repeating name twice, we do

    PACKAGE-UPGRADED: ${ name }: ${ from } -> ${ to }

Reply:
Removes the redundant name output. I like it.